### ISSUE TITLE
Question about mixed logit model (sigma)

### ISSUE THREAD

**Issue by pradiptamega** [SEN1221TUD/Q2_2024#25]
I have a few questions about determining the sigma value in MXL. Specifically, about its sign convention. Is it typically negative or positive? 

Additionally, I've noticed that the initial value for sigma can impact the final results in the exam practice case. For instance, starting with 1 or -1 can lead to different outcomes and signage of sigma value, and slightly affect other paramters as well. 

Could you provide some explanation for that? Thank you for your attention to this.

**Comment by sandervancranenburgh** [comment]
Hi, if you are referring to a normal distribution, then the sign of sigma is inconsequential. Switching the sign of sigma will only flip the distribution around its y-axis. But the normal distribution is symmetric. So this will not have any impact.

**Comment by sandervancranenburgh** [comment]
Yes, but that is because of the low number of draws. Did you see the plots at the end of lab 2B, where the VTT is plotted as a function the no. draws?

**Comment by pradiptamega** [comment]
Ah, thank you for the explanation, I think I understand now.
